doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -538,7 +538,43 @@ \subsubsection{Inputs}\label{inputs-10-021}
 
 \paragraph{Field: Ventilation Control Mode Schedule Name}\label{field-ventilation-control-mode-schedule-name}
 
-The name of a schedule defined elsewhere in the input file. This schedule determines whether or not for a given time the hybrid ventilation control mode is to be applied. Schedule values equal to zero indicate no ventilation control, resulting in natural ventilation and HVAC system operation being performed based on their own controls. Schedule values equal to one denote temperature control for either cooling or heating, which is determined internally based on thermostat set point and temperature control type. The temperature control is restricted between the minimum and maximum outdoor temperatures provided in two additional input fields (below). Schedule values equal to two denote enthalpy control, which is restricted between minimum and maximum outdoor enthalpy values given in two additional input fields (below). Schedule values equal to three denote dewpoint control for either dehumidification or humidification. Schedule values equal to four represent outdoor ventilation air control. The outdoor ventilation air control works with AirflowNetwork opening objects only, and is not allowed to work with ZoneVentilation:* and ZoneMixing objects.
+The name of a schedule defined elsewhere in the input file. This schedule determines whether or not for a given time the hybrid ventilation control mode is to be applied. The integer values are required in the schedule and should be varied between 0 and 7. The following bullets show each schedule value and its corresponding control mode:
+
+\begin{description}
+  \item
+  0 = No ventilation control
+  
+  Natural ventilation and HVAC system operation are performed based on their own controls.
+  \item
+  1 = Temperature control for either cooling or heating
+  
+  The control mode is determined internally based on thermostat set point and temperature control type and is restricted between the minimum and maximum outdoor temperatures provided in two additional input fields: Minimum Outdoor Temperature and Maximum Outdoor Temperature.
+  \item
+  2 = Enthalpy control
+  
+  The control is restricted between minimum and maximum outdoor enthalpy values given in two additional input fields: Minimum Outdoor Enthalpy and Maximum Outdoor Enthalpy.
+  \item
+  3 = Dewpoint control for either dehumidification or humidification
+  \item
+  4 = Outdoor ventilation air control
+  
+  The outdoor ventilation air control works with AirflowNetwork opening objects only, and is not allowed to work with ZoneVentilation:* and ZoneMixing objects.
```

Comment:
@mjwitte Thanks for more review. I am going to add more explanation of Types 3 and 4.